FDijkmans.md

```diff
@@ -0,0 +1 @@
+Example txt
```

Comment:
@FDijkmans Good start so far! Maybe try some markdown? Try to make something **bold** or *italic* or a [link](https://guides.github.com/features/mastering-markdown/)